src/main/java/seedu/tasklist/TaskList.java

```diff
@@ -52,21 +52,8 @@ public void changeLocation(int taskIndex, String newLocation) {
     public void changeReminder(int taskIndex, String newReminder) {
         taskList.get(taskIndex).setReminder(newReminder);
     }
-
-    /**
-     * Change category of a task and change the category mapping.
-     *
-     * @param taskIndex   Index of task that needs to be changed.
-     * @param newCategory Newly category.
-     */
-    public void changeCategory(int taskIndex, String newCategory) {
-        String oldCategory = taskList.get(taskIndex).getCategory();
-        this.categoryMap.get(oldCategory).remove(taskIndex);
-        taskList.get(taskIndex).setCategory(newCategory);
-        updateCategoryMap(newCategory, taskIndex);
-    }
-
-
+  
+  //Other methods:
```

Comment:
The style error seems to be here because of the indentation of the "//Other methods" comment

Reply:
![image](https://user-images.githubusercontent.com/49268567/78962164-55ff7480-7b26-11ea-8d37-85c2a564ddf5.png)
Interestingly, all the constants cannot be recognized. I am totally confused.